api-contracts/openapi/paths/webhook-worker/webhook-worker.yaml

```
@@ -121,3 +121,45 @@ webhookworker:
             schema:
               $ref: "../../components/schemas/_index.yaml#/APIErrors"
         description: Method not allowed
+
+webhookworkerRequests:
+  get:
+    description: Lists all requests for a webhook
+    summary: List webhook requests
+    operationId: webhook:listRequests
```

```suggestion
    operationId: webhook-requests:list
```

Just to keep it consistent, allowed verbs in the second slot are `get`, `list`, `create`, `update`, `delete` (will make rbac much easier eventually)